pages/common/zfs.md

```diff
@@ -1,6 +1,7 @@
 # zfs
 
 > Manage ZFS filesystems.
+> More information: <https://docs.oracle.com/cd/E23823_01/html/819-5461/zfsover-2.html>
```

Comment:
I wasn't sure about this one, would like to hear someone's thoughts. The man page says

> For information about using the ZFS web-based management tool and other ZFS features, see the Solaris ZFS Administration Guide.

So that's what I decided to link -- but maybe the man page itself would be more helpful.

Reply:
I don't like it, but I can't really find anything better.

I agree that the manpage would be more useful, so that would be a good option.